src/localization/authStrings.json

```diff
@@ -39,6 +39,17 @@
     "password": "ລະຫັດຜ່ານ",
     "user_name": "ຊື່​ຜູ້​ນຳ​ໃຊ້"
   },
+  "my": {
+    "logging_in": "Login ဝင်ရောက်နေသည်",
+    "login": "Login ဝင်ရောက်သည်",
```

Comment:
the `Login` isn't required, afaik

Reply:
You sure? Took it directly from the the translations... maybe should confirm with @AungHtunOo88?

Reply:
We just started using standard Unicode for our country recently. So, some google translation may not come up perfectly but with general usage (e.g - log-in). But, we can confirm that our translation will be easily understood by the basic user.